.github/workflows/build.yml

```diff
@@ -0,0 +1,250 @@
+name: build
+
+on:
+  push:
+    # Sequence of patterns matched against refs/heads
+    branches:
+    # Provide the release branch
+      - crt_release/1.9.x
```

Comment:
Should this be `release/1.9.x` so that it matches our release branch name?

Reply:
Yes updated to `release/1.9.x` (kept this to trigger the current branch releases to validate that all the updates are passing)